app/src/main/kotlin/co/netguru/baby/monitor/client/application/ServiceBindingsModule.kt

```diff
@@ -3,6 +3,7 @@ package co.netguru.baby.monitor.client.application
 import co.netguru.baby.monitor.client.feature.babycrynotification.BabyMonitorMessagingService
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/feb0c51d1649c186c776faa50e562320ab9dc395/app/src/main/kotlin/co/netguru/baby/monitor/client/application/ServiceBindingsModule.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>